guides/plugins/plugins/plugins-for-symfony-developers.md

```diff
@@ -1,2 +1,64 @@
 # Plugins for Symfony developers
 
+## Overview
+
+This guide serves as an entry point for developers, who are familiar with the concepts of `Symfony bundles`.
+
+## Prerequisites
+
+This guide handles some base concepts of Shopware plugins. 
+You may want to have a look at [plugin base guide](./plugin-base-guide.md) first.
+
+As this guide also references the functionality of Symfony bundles,
+you should have at least basic knowledge of it.
+You may want to have a look or refresh your knowledge with help of Symfony's [bundle documentation](https://symfony.com/doc/current/bundles.html).
+
+## Symfony bundles
+
+A bundle is the Symfony's preferred way to provide additional third-party features to any Symfony application.
+Those bundles are everywhere: Symfony even outsources many of its core features into external bundles.
+The template engine `Twig`, the `Security` bundle, the `WebProfiler`, 
+as well as many other third-party bundles can be installed on demand to extend your Symfony application in any way.
+The Bundle System is Symfony's way of providing an extendable framework with plugin capabilities.
+
+## Shopware plugins
+
+Shopware builds upon the `Symfony Bundle System` to extend its functionality even more.
```

Comment:
```suggestion
Shopware is building on the `Symfony Bundle System` to extend its functionality even more.
```

Not sure about that though, but it's the most common result if you search with deepl and google.